src/layouts/Header.jsx

```diff
@@ -1,11 +1,37 @@
 import React from 'react';
-import { useAuth } from '../context/AuthProvider';
+import logoImage from '../assets/images/logo.png';
+import Search from '../components/Search';
+import NavBar from '../components/NavBar';
+import { useNavigate } from 'react-router-dom';
 
 const Header = () => {
-  const { isLoggedIn } = useAuth();
+  const navigate = useNavigate();
+
+  const handleHomeNavigation = () => {
+    navigate('/');
+  };
 
   return (
-    <div>{isLoggedIn ? 'You are logged in' : 'You are not logged in'}</div>
+    <div className="p-5">
+      <div className="flex items-center justify-between">
+        <div className="flex items-center" onClick={handleHomeNavigation}>
+          <img
+            className="h-[26px] w-[33px] sm:h-[45px] sm:w-[60px]"
+            src={logoImage}
+          />
+          <h1 className="font-headings sm:text-2xl sm:font-bold">Re-Books</h1>
```

Comment:
In the [Figma design](https://www.figma.com/design/rrsSHLd7U2oiTuWdhz9Kzh/Books?node-id=235-354&node-type=frame&t=749gIs9WhlOgbzwF-0), instead of a dash (–), there’s a colon (:): **Re:Books**.